fuzztests/runFuzzTests.js

```diff
@@ -47,7 +47,8 @@ async function executeFuzzTest(file, name) {
 		});
 		test.on("close", (code) => {
 			console.log(`--- Finished fuzz test ${file} > ${name} with code ${code}`);
-			resolve();
+			if (code !== 0 && code !== null) reject(code);
```

Comment:
Could be `if(!code)`.

Reply:
We can't do that: `!code` is true when `code === 0`

Reply:
Ah, sure. Mixed up the semantics of the return codes.